utils/imagenarator.py

```diff
@@ -0,0 +1,40 @@
+from PIL import Image, ImageDraw,ImageFont 
+import textwrap
+import re
+def draw_multiple_line_text(image, text, font, text_color,padding):
+    '''
+    Draw multiline text over given image
+    '''
+    draw = ImageDraw.Draw(image)
+    Fontperm= font.getsize(text)
+    image_width, image_height = image.size
+    lines = textwrap.wrap(text, width=50)
+    y=(image_height/2)-(((Fontperm[1]+(len(lines)*padding)/len(lines))*len(lines))/2)
+    for line in lines:
+        line_width, line_height = font.getsize(line)
+        draw.text(((image_width - line_width) / 2, y), 
+                    line, font=font, fill=text_color)
+        y += line_height + padding
+
+
+def imagemaker( theme,reddit_obj,
+                txtclr,
+                padding=5
+                ):
+    texts=reddit_obj['thread_post']
+    id = re.sub(r"[^\w\s-]", "", reddit_obj["thread_id"])
+    font=ImageFont.truetype("C:\\fonts\\NotoSans-Regular.ttf", 20)
```

Comment:
Why is this an absolute path that only works on the C drive? Maybe get the path: https://stackoverflow.com/questions/51520/how-to-get-an-absolute-file-path-in-python